tests/api/transactions/GET.test.js

```diff
@@ -77,4 +80,18 @@ describe("GET /api/transactions", () =>{
       expect(res.status).toEqual(200);
     });
   });
+  describe("user id does not have a transaction that pertain to it", ()=>{
+    it("should return error object", async() =>{
+      await expect(async() => {
+        // await searchQueryTrans();
+        const res = await request(app)
+          .get("/api/transactions")
+          .set("Accept", "application/json")
+          .set("Cookie", [wrongIdCookie]);
+      }).rejects.toMatchObject({
+        statusCode: 500,
+        message: "orm tool failed"
+      });
```

Comment:
If no transactions for the user, we should expect an empty array.

Reply:
Resolved
https://github.com/The-Silent-Corner/sad-twt/pull/49/commits/a6e5a42d475d0f017602ac0b399cd9dfde4687c1#diff-24fa6350a0aad2d067849b46fd2b2f506df6ed12d0d10d23fca506ad58444e72R88